### PR TITLE
Bomberman implementation

### PR DESCRIPTION
**Pull request by Abergard**
Not finished yet.
Only a header file and empty test. Preparation for working with it.

---
This change is [Reviewable](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73)

### CODE REVIEW

File: src/game/Bomberman.hpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#include "HumanPlayer.hpp"
+
+class Bomberman : public Entity
+{
+public:
+    void update(DeltaTime) override;
+    bool areYouDead() const override;
+
+private:
+    physics::PhysicsId physicsId;
```
Comment (moose-x86) [inline src/game/Bomberman.hpp]: consider rearranging membres in way 
- normal members
- unique_ptrs
- shared_ptrs

**Review by moose-x86 — APPROVED** [review]

**Comment by kn65op** [comment]
Review status: 0 of 4 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[src/game/Bomberman.hpp, line 20 at r1](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/73#-L4FSB-rFnIKeKNKTr0r-r1-20:-L4qITkgKdtoqVX3LN-0:bb74njr) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/1c03e84e9fa494acfd8a037ba60ff3fb60559ebe/src/game/Bomberman.hpp#L20)):*
<details><summary><i>Previously, moose-x86 (Przemek) wrote…</i></summary><blockquote>

consider rearranging membres in way 
- normal members
- unique_ptrs
- shared_ptrs

</blockquote></details>

Test

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/73)*

File: src/game/Bomberman.hpp (hunk 1 above)
Comment (kn65op) [reply src/game/Bomberman.hpp]: Test3

**Comment by Abergard** [comment]
Taks has been finished and it is ready for review :)

**Comment by kn65op** [comment]
Reviewed 19 of 21 files at r2.
Review status: 18 of 19 files reviewed at latest revision, 1 unresolved discussion.

---

*[CMakeLists.txt, line 4 at r2](https://beta.reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73#-LARq5Ah-I7wJuJtUzQ0:-LARq5Ah-I7wJuJtUzQ1:b-slmrw7) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/2f953ea5e9b7de7aa14773ca4b8ff8f7ee1b8756/CMakeLists.txt#L4)):*
> ```CMake
> project("bomberman" CXX)
> 
> find_package(Boost 1.6 REQUIRED)
> ```

Shouldn't  be 1.65?

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73)*

**Comment by kn65op** [comment]
:lgtm:

---

Reviewed 2 of 21 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73#-:-LARrg_oEUF-JumnyDYL:bnfp4nl)*

**Comment by Abergard** [comment]
Review status: 16 of 19 files reviewed at latest revision, 2 unresolved discussions.

---

*[CMakeLists.txt, line 4 at r2](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73#-LARq5Ah-I7wJuJtUzQ0:-LEOCJeyBh3kSGynOOM0:bgf45pw) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/2f953ea5e9b7de7aa14773ca4b8ff8f7ee1b8756/CMakeLists.txt#L4)):*
<details><summary><i>Previously, kn65op (Tomasz Drzewiecki) wrote…</i></summary><blockquote>

Shouldn't  be 1.65?
</blockquote></details>

It is only a minimal version needed to compile. If we want to specify a specific version it is job for another task.

---

*[src/game/Bomberman.hpp, line 20 at r1](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73#-L4FSB-rFnIKeKNKTr0r-r1-20:-LEODmX57FwkLVGA4EAJ:b-896fix) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/1c03e84e9fa494acfd8a037ba60ff3fb60559ebe/src/game/Bomberman.hpp#L20)):*
<details><summary><i>Previously, kn65op (Tomasz Drzewiecki) wrote…</i></summary><blockquote>

Test
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73)*

**Comment by moose-x86** [comment]
Reviewed 16 of 19 files at r2, 3 of 3 files at r3.
Review status: all files reviewed, 2 unresolved discussions (waiting on @moose-x86 and @kn65op)

---



*Comments from [Reviewable](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73)*

**Review by kn65op — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/73)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**